libvast/test/view.cpp

```diff
@@ -45,6 +45,46 @@ TEST(container view) {
   CHECK_EQUAL(i - j, xs.size() - 1);
 }
 
+TEST(set view) {
+  auto xs = set{42, true};
+  auto v = make_view(xs);
+  REQUIRE_EQUAL(v->size(), xs.size());
+  auto i = v->begin();
+  // FIXME: we currently use a std::set instead of a steady set, which relies
+  // on a non-deterministic ordering. This unit test may fail on some platforms
```

Comment:
> non-deterministic ordering

`std::set` does have a deterministic order. Are you referring to `std::unordered_set`? Otherwise all platforms must have the same order unless `operator<` for the values is somehow messed up.

Reply:
Yea, `std::set::operator<` is deterministic. But using `std::type_index` for the implementation of `operator<` is not. 😉 

The comment was admittedly not very clear.